ubg/src/pages/Room.js

```diff
@@ -413,7 +413,8 @@ function Room({setUsersData, setCurrentUser, setRoomData}) {
                       </div>
                       <Chat
                         disabled={(mafiaChatSelected && roomData.day) ||
-                          !inGame}
+                          !inGame ||
```

Comment:
Will this disable the normal chat for mafia during the night as well?

Reply:
I don't think so cuz !mafiaChatSelected make sure that it's not mafia chat